dice_ml/explainer_interfaces/explainer_base.py

```diff
@@ -3,14 +3,22 @@
    All methods are in dice_ml.explainer_interfaces"""
 
 import numpy as np
+import pandas as pd
+import random
+import timeit
+import copy
+
+from dice_ml import diverse_counterfactuals as exp
 
 class ExplainerBase:
 
-    def __init__(self, data_interface):
+    def __init__(self, data_interface, model_interface):
```

Comment:
you can make this `model_interface=None` to maintain compatibility with old code and avoid pytest errors that you see in this PR.